index.js

```diff
@@ -11,26 +11,50 @@
 let store = {};
 
 const _callsubscribers = (key, newValue, prevValue) => {
-  store[key].subscribers.forEach(callback => {
-    if (typeof callback === 'function') {
-      callback(newValue, prevValue);
+  store[key].subscribers.forEach(config => {
+    if (!config.options.onlyOnChange || !deepEqual(newValue, prevValue)) {
+      config.callback(newValue, prevValue);
     }
   });
 };
 
+// Thanks to https://stackoverflow.com/a/25456134/1954789
+function deepEqual(x, y) {
```

Comment:
We should add a lot of tests for this function.
I mean, tests checking if this function is checking properly that the parameters (objects, arrays, strings, bool, numbers, etc) are deep equals.

Reply:
Yes, didn't have the time... I will implement a test on objects and array's but to test all cases is a lot of work... could it be done later?

Reply:
Delivered